<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioEditor</class>
 <widget class="QWidget" name="AudioEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditTitle"/>
    </item>
    <item>
     <widget class="QLabel" name="labelPath_2">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditDescription"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButtonBrowse">
        <property name="text">
         <string>Browse</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/resources/resources/browse.png</normaloff>:/new/resources/resources/browse.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonPlay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/resources/resources/play.png</normaloff>:/new/resources/resources/play.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonPause">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Pause</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/resources/resources/pause.png</normaloff>:/new/resources/resources/pause.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonStop">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/resources/resources/stop.png</normaloff>:/new/resources/resources/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Phonon::VolumeSlider" name="volumeSlider"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/resources/resources/save.png</normaloff>:/new/resources/resources/save.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/new/resources/resources/delete.png</normaloff>:/new/resources/resources/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
